sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj

```diff
@@ -7,13 +7,15 @@
 		<PackageId>Microsoft.Azure.Management.Sql</PackageId>
 		<Description>Azure SQL Management SDK library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Sql</AssemblyName>
-		<Version>1.43.0-preview</Version>
+		<Version>1.44.0-preview</Version>
 		<PackageTags>Microsoft Azure SQL Management;SQL;SQL Management;</PackageTags>
 		<PackageReleaseNotes>
 			<![CDATA[
 New features:
 - Added private link connection support for data sync clients
 - Added support for Azure SQL Managed Instance failover operation
```

Comment:
You don't need to keep release notes from last published release

Reply:
Got it. I will re-sync with master when this is ready to merge in and only include MY release notes in this file.

Reply:
Updated.